config/webpack.config.js

```diff
@@ -481,6 +481,11 @@ module.exports = async (env = {}) => {
             from: 'src/site/assets',
             to: buildPath,
           },
+          {
+            from:
+              'node_modules/@department-of-veterans-affairs/component-library/dist/assets',
+            to: `${buildPath}/assets`,
```

Comment:
Ah nice. This was the part I had wrong in my own research. I think I was specifying `src/assets` or something hard coded instead of `buildPath`